src/bin/wasm-tools/component.rs

```diff
@@ -202,6 +209,7 @@ impl WitResolve {
 
     fn load(&self) -> Result<(Resolve, Vec<PackageId>)> {
         let mut resolve = Self::resolve_with_features(&self.features);
+        resolve.all_features = self.all_features;
```

Comment:
Could setting this field be pushed into `resolve_with_features`? That way other callers don't have to worry about processing the argument.

Reply:
Does WIT define the value constraints for `feature =`? Could `--all-features` alias `--features *`?

Reply:
Where can I find the formal definition of `id`?

Reply:
Sorry not sure I quite understand, can you elaborate on what you meant by value constraints? Also what `id` are you referring to?

As for `--features *` being the same as `--all-features`, it should be relatively easy to implement but it might be difficult for shells typically because `*` by default expands to all the files in the current directory.

Reply:
I just pushed a change that makes `--all-features` an alias for `--features *`, since `id` must be a lower kebab-case identifier.

Reply:
`*` isn’t great for shell expansion—would `--features .` be acceptable?